src/test/java/org/broadinstitute/dsm/QuartzExternalShipperTest.java

```diff
@@ -64,7 +64,7 @@ public static void last() {
     public void externalShipperJob() throws Exception {
 //        uploadKit();
 
-        ExternalShipper shipper = (ExternalShipper) Class.forName(DSMServer.getClassName("gbf")).newInstance();//GBFRequestUtil
+        GBFRequestUtil shipper = new GBFRequestUtil();
```

Comment:
No need to instantiate via reflection, since we know exactly the `GBFRequestUtil` class we're testing here.